http/daemon.go

```diff
@@ -32,7 +33,8 @@ type Daemon struct {
 }
 
 var (
-	connectionDuration = prometheus.NewGaugeFrom(stdprometheus.GaugeOpts{
+	ErrEndpointDeprecated = errors.New("Your fluxd version is deprecated - please upgrade")
```

Comment:
Could use `ErrorDeprecated` from `http/errors.go` here, but it is multiline (IMO not great for printing to a daemon log) and mentions fluxctl specifically - thoughts?